app/src/Homepage/Homepage.css

```diff
@@ -46,3 +46,78 @@
   height: 200vh;
   text-align: center;
 }
+
+/* Welcome header */
+.welcome {
+  text-align: center;
+  margin-top: 60px;
+  font-size: 300%;
+}
+
+/* How it works: header */
+.howitworks {
+  position: absolute;
+  top: 15%;
+  height: 50%;
+  width: 100%;
+}
+
+/* Explanation below the How it works: header */
+.explanation {
+  font-weight: normal;
+}
+
+/* The rating boxes for difficulty, workload, and practicality */
+.ratingbox {
+  font-weight: bold;
+  font-size: 130%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  position: absolute;
+  margin-top: 20px;
+  left: 10%;
+  height: 10%;
+  width: 14%;
```

Comment:
Resize recommendation:

Probably add a minimum width so that the words are always contained within the boxes

Reply:
Fixed